sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.sources=.
 sonar.tests=.
 sonar.test.inclusions=**/*.spec.ts
 sonar.exclusions=**/*.js,jest.config.ts,globalSetup.ts,globalTeardown.ts,**/*.app.ts,**/seed-data/*.ts
-sonar.coverage.exclusions=**/board-management.uc.ts,**/*.module.ts,**/*.factory.ts
+sonar.coverage.exclusions=**/modules/tldraw/**/*.ts,**/board-management.uc.ts,**/*.module.ts,**/*.factory.ts

```

Comment:
Why is it exluded from code coverage?

Reply:
We will change whole directory, for specific files. As me and Dawid Wasik tried, we could't do tests for every line in those files and as i understand sonar requires 100% coverage

Reply:
Yes it's required and not optional and you should not try to slip out of it.
@CeEv @Metauriel  cann you say somthing about it?